plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemPotion.java

```diff
@@ -342,21 +347,21 @@ public static void register() {
             }
             if (attribute.startsWith("is_splash", 2)) {
                 attribute.fulfill(1);
-                return new ElementTag(object.item.getBukkitMaterial() == Material.SPLASH_POTION);
+                return new ElementTag(object.object.getBukkitMaterial() == Material.SPLASH_POTION);

```

Comment:
`object.getMaterial()`

Reply:
Oops, was updating old usages and missed that - fixed